packages/i18n/src/cs.ts

```diff
@@ -0,0 +1,74 @@
+    checkItems:     (issue) => `Neplatná položka: Obdrženo ${issue.received}`,
+    creditCard:     (issue) => `Neplatná hodnota kreditní karty: Obdrženo ${issue.received}`,
+    cuid2:          (issue) => `Neplatné Cuid2: Obdrženo ${issue.received}`,
+    decimal:        (issue) => `Neplatné desetinné číslo: Obdrženo ${issue.received}`,
+    digits:         (issue) => `Neplatné číslice: Obdrženo ${issue.received}`,
+    email:          (issue) => `Neplatný email: Obdrženo ${issue.received}`,
+    emoji:          (issue) => `Neplatné emoji: Obdrženo ${issue.received}`,
+    empty:          (issue) => `Neplatná délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    endsWith:       (issue) => `Neplatný konec: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    everyItem:      (issue) => `Neplatná položka: Obdrženo ${issue.received}`,
+    excludes:       (issue) => `Neplatný obsah: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    finite:         (issue) => `Neplatné konečné číslo: Obdrženo ${issue.received}`,
+    hash:           (issue) => `Neplatný hash: Obdrženo ${issue.received}`,
+    hexColor:       (issue) => `Neplatná hexadecimální barva: Obdrženo ${issue.received}`,
+    hexadecimal:    (issue) => `Neplatné hexadecimální číslo: Obdrženo ${issue.received}`,
+    imei:           (issue) => `Neplatné IMEI: Obdrženo ${issue.received}`,
+    includes:       (issue) => `Neplatný obsah: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    integer:        (issue) => `Neplatné celé číslo: Obdrženo ${issue.received}`,
+    ip:             (issue) => `Neplatná IP adresa: Obdrženo ${issue.received}`,
+    ipv4:           (issue) => `Neplatná IPv4 adresa: Obdrženo ${issue.received}`,
+    ipv6:           (issue) => `Neplatná IPv6 adresa: Obdrženo ${issue.received}`,
+    isoDate:        (issue) => `Neplatné datum: Obdrženo ${issue.received}`,
+    isoDateTime:    (issue) => `Neplatné datum a čas: Obdrženo ${issue.received}`,
+    isoTime:        (issue) => `Neplatný čas: Očekáváno "hh:mm", ale obdrženo ${issue.received}`,
+    isoTimeSecond:  (issue) => `Neplatný čas: Očekáváno "hh:mm:ss", ale obdrženo ${issue.received}`,
+    isoTimestamp:   (issue) => `Neplatný časový údaj: Očekáván ISO 8601 formát, ale obdrženo ${issue.received}`,
+    isoWeek:        (issue) => `Neplatný týden: Obdrženo ${issue.received}`,
+    length:         (issue) => `Neplatná délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    mac:            (issue) => `Neplatná MAC adresa: Obdrženo ${issue.received}`,
+    mac48:          (issue) => `Neplatná 48-bitová MAC adresa: Obdrženo ${issue.received}`,
+    mac64:          (issue) => `Neplatná 64-bitová MAC adresa: Obdrženo ${issue.received}`,
+    maxBytes:       (issue) => `Překočena maximální v bajtech (bytes): Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
```

Comment:
- typo in Překočena -> Překročena
- I'd rather prefer: Překročena maximální hodnota v bajtech. otherwise it has no sense, because there is not clear what is "prekroceno maximalni v bytech"

Reply:
fixed